server/index.js

```diff
@@ -10,7 +10,7 @@ require("./database/client").checkConnection();
 const app = require("./app/config");
 
 // Get the port from the environment variables
-const port = process.env.APP_PORT;
+const port = process.env.APP_PORT || 3000;
```

Comment:
please d'ont, it will conflict with your team default, running the frontend on 3000